prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -2224,25 +2205,9 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
 
         let slice_ty_ref = self.mir_encoder.get_operand_ty(&args[0]);
         let slice_ty = if let ty::TyKind::Ref(_, slice_ty, _) = slice_ty_ref.kind() { slice_ty } else { unreachable!() };
-        let st = self.encode_slice_types(slice_ty).with_span(span)?;
-        let slice_len_name = self.encoder.encode_builtin_function_use(
-            BuiltinFunctionKind::SliceLen {
-                slice_ty_pred: st.slice_pred.clone(),
-                elem_ty_pred: st.elem_pred,
-            }
-        );
+        let st = self.encoder.encode_slice_types(slice_ty, None).with_span(span)?;
```

Comment:
Please replace `st` and `at` with a more descriptive name. We know what they are now but … :)